docs/en/flight_controller/radiolink_pix6.md

```diff
@@ -0,0 +1,346 @@
+| 3   | DP      | +3.3V |
+| 4   | GND     | GND   |
+
+### SWITCH
+
+| Pin | Signal         | Volt  |
+| --- | -------------- | ----- |
+| 1   | VCC            | +3.3V |
+| 2   | !IO_LED_SAFETY | GND   |
+| 3   | SAFETY         | GND   |
+
+### Buzzer port
+
+| Pin | Signal  | Volt |
+| --- | ------- | ---- |
+| 1   | VCC     | +5V  |
+| 2   | BUZZER- | +5V  |
+
+### Spektrum/DSM Port (PH1.25-3P)
+
+| Pin | Signal | Volt  |
+| --- | ------ | ----- |
+| 1   | VCC    | +3.3V |
+| 2   | GND    | GND   |
+| 3   | Signal | +3.3V |
+
+### Debug port (SH1.0-8P)
+
+| Pin | Signal    | Volt  |
+| --- | --------- | ----- |
+| 1   | VCC       | +5V   |
+| 2   | FMU_SWCLK | +3.3V |
+| 3   | FMU_SWDIO | +3.3V |
+| 4   | TX(UART7) | +3.3V |
+| 5   | RX(UART7) | +3.3V |
+| 6   | IO_SWCLK  | +3.3V |
+| 7   | IO_SWDIO  | +3.3V |
+| 8   | GND       | GND   |
+
+## Building Firmware
+
+To [build PX4](../dev_setup/building_px4.md) for this target:
+
+```sh
+make radiolink_PIX6_default
+```
+
+## Installing PX4 Firmware
+
+The firmware can be installed in any of the normal ways:
+
+- Build and upload the source
+
+  ```sh
+  make radiolink_PIX6_default upload
+  ```
+
+- [Load the firmware](../config/firmware.md) using _QGroundControl_.
+  You can use either pre-built firmware or your own custom firmware.
+
+  ::: info
+  At time of writing the only pre-built software is `PX4 main` (see [Installing PX4 Main, Beta or Custom Firmware](../config/firmware.md#installing-px4-main-beta-or-custom-firmware)).
+  Release builds will be supported for PX4 v1.17 and later.
+  :::
+
+## PX4 Configuration
+
+In addition to the [basic configuration](../config/index.md), the following parameters are important:
+
+| Parameter                                                            | Setting                                                                                                                 |
+| -------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| [SYS_HAS_MAG](../advanced_config/parameter_reference.md#SYS_HAS_MAG) | This should be disabled since the board does not have an internal mag. You can enable it if you attach an external mag. |
+
+### Powering the PIX6
+
+The PIX6 has 2 dedicated power monitor ports, each with a 6 pin connector.
+One is the Analog power monitor (`POWER1`), and the others is the I2C power monitor (`POWER2`).
+
+The power module that comes with the flight controller with a wide voltage input range of 2-12S (7.4-50.4V), a maximum detection current of 90A (single ESC maximum detection current is 22.5A), a BEC output voltage of 5.3±0.2V, and a BEC output current of 2A.
+
+![Radiolink power modules MODULES](../../assets/flight_controller/radiolink_pix6/radiolink_power_modules.png)
+
+The PIX6 also supports power modules from other manufacturers, such as [holybro_pm02d](../power_module/holybro_pm02d.md).
```

Comment:
FYI I moved this into a generic powering section. You don't link to any power modules for purchase (and state that one is supplied), so no need to put as recommended.